sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -51,26 +51,31 @@ import (
 // TODO(herohde) 5/16/2017: the Dataflow flags should match the other SDKs.
 
 var (
-	endpoint             = flag.String("dataflow_endpoint", "", "Dataflow endpoint (optional).")
-	stagingLocation      = flag.String("staging_location", "", "GCS staging location (required).")
-	image                = flag.String("worker_harness_container_image", "", "Worker harness container image (required).")
-	labels               = flag.String("labels", "", "JSON-formatted map[string]string of job labels (optional).")
-	serviceAccountEmail  = flag.String("service_account_email", "", "Service account email (optional).")
-	numWorkers           = flag.Int64("num_workers", 0, "Number of workers (optional).")
-	maxNumWorkers        = flag.Int64("max_num_workers", 0, "Maximum number of workers during scaling (optional).")
-	diskSizeGb           = flag.Int64("disk_size_gb", 0, "Size of root disk for VMs, in GB (optional).")
-	diskType             = flag.String("disk_type", "", "Type of root disk for VMs (optional).")
-	autoscalingAlgorithm = flag.String("autoscaling_algorithm", "", "Autoscaling mode to use (optional).")
-	zone                 = flag.String("zone", "", "GCP zone (optional)")
-	network              = flag.String("network", "", "GCP network (optional)")
-	subnetwork           = flag.String("subnetwork", "", "GCP subnetwork (optional)")
-	noUsePublicIPs       = flag.Bool("no_use_public_ips", false, "Workers must not use public IP addresses (optional)")
-	tempLocation         = flag.String("temp_location", "", "Temp location (optional)")
-	machineType          = flag.String("worker_machine_type", "", "GCE machine type (optional)")
-	minCPUPlatform       = flag.String("min_cpu_platform", "", "GCE minimum cpu platform (optional)")
-	workerJar            = flag.String("dataflow_worker_jar", "", "Dataflow worker jar (optional)")
-	workerRegion         = flag.String("worker_region", "", "Dataflow worker region (optional)")
-	workerZone           = flag.String("worker_zone", "", "Dataflow worker zone (optional)")
+	endpoint               = flag.String("dataflow_endpoint", "", "Dataflow endpoint (optional).")
+	stagingLocation        = flag.String("staging_location", "", "GCS staging location (required).")
+	image                  = flag.String("worker_harness_container_image", "", "Worker harness container image (required).")
```

Comment:
```suggestion
	image                  = flag.String("worker_harness_container_image", "", "Worker harness container image (optional).")
```

Definitely optional at this stage. I haven't used the flag in so long it doesn't show up in my command line history.